src/components/Order_page/index.js

```diff
@@ -0,0 +1,26 @@
+import {useEffect, useState} from "react"
+import Item_card from "../Item_card"
+import './Order_page.scss'
+
+const Order_page = ({activeRestaurantId}) => {
+    const [foodItemData, setItemData] = useState([])
+
+    useEffect( () =>{
+    fetch(`http://localhost:8080/restaurants/${activeRestaurantId}`)
+        .then(res => res.json())
+        .then(function (data) {
```

Comment:
Use a fat arrow for this callback as well to be consistent with the first then block